tonic/models/vic/vic.py

```diff
@@ -67,8 +67,17 @@ def run(self, global_param, logdir=None):
 
         return self.returncode
 
-    def _call_vic(self, *args):
-        vic_args = [self.executable] + [a for a in args]
+    def _call_vic(self, mpi_proc=None, *args):
+
+        vic_args = []
+
+        if mpi_proc is not None:
+            if isinstance(mpi_proc, int):
+                raise TypeError("number of processors must be specified as an integer")   
+ 
+            vic_args.extend(['mpirun', '-np', '0:d}'.format(mpi_proc)])  
```

Comment:
@dgergel - can we fix this little bug.  You're missing a `{` in the format statement.  This is now causing test to fail in VIC.